guillotina/contrib/dbusers/content/users.py

```diff
@@ -57,10 +57,17 @@ class IUser(IFolder, IPrincipal):
 class User(Folder):
     username = email = name = password = None
     disabled = False
-    user_roles = ["guillotina.Member"]
-    user_groups: typing.List[str] = []
-    user_permissions: typing.List[str] = []
-    properties: typing.Dict[str, typing.Any] = {}
+    user_roles = None
+    user_groups: typing.Optional[typing.List[str]] = None
+    user_permissions: typing.Optional[typing.List[str]] = None
+    properties: typing.Optional[typing.Dict[str, typing.Any]] = None
```

Comment:
Just wondering: whats the point of defining the class attributes? These attributes are created dynamically from the content-type schema, right? If its for the mypy annotations, now that the attributes are defined in the __init__, we can define the annotations there.

(Same for Group contenttype)

Reply:
Thought it's something inherited from the initial implementation, agree on clean them.